### PR TITLE
fixed typo

### PR DESCRIPTION
**Pull request by mitchnegus**

### CODE REVIEW

File: statsmodels/datasets/statecrime/data.py [hunk 1]
```diff
@@ -46,7 +46,7 @@
     single
         Calculated from 2009 1-year American Community Survey obtained obtained
         from Census. Variable is Male householder, no wife present, family
-        household combined with Female household, no husband prsent, family
+        household combined with Female householder, no husband prsent, family
```
Comment (jbrockmendel) [inline statsmodels/datasets/statecrime/data.py]: prsent --> present

**Comment by jbrockmendel** [comment]
@josef-pkt easy as it gets

**Comment by jbrockmendel** [comment]
@josef-pkt @ChadFulton 4 line diff, just fixing typos in docstrings.

This kind of thing should be delegated so that this talented contributor gets a dose of Warm Glow, is motivated to keep up the good work.

**Comment by josef-pkt** [comment]
@jbrockmendel  you start to get annoying with the same comment in every PR

**Comment by jbrockmendel** [comment]
Thanks for the heads up.  Please don't take it out on this contributor.

**Comment by mitchnegus** [comment]
sorry, haven't been a user long enough to have substantive additions just yet... maybe soon!

**Comment by jbrockmendel** [comment]
@mitchnegus you’re doing great.  Having a fresh pair of eyes look at the code is really valuable.  Keep it up!

**Comment by jbrockmendel** [comment]
@mitchnegus FWIW this has has been ported over and merged in [sm2](https://github.com/jbrockmendel/sm2)

**Comment by jbrockmendel** [comment]
@thequackdaddy 2 line diff as obvious as it was weeks ago...

**Pull request merged**